app/assets/stylesheets/application.css.scss

```diff
@@ -44,7 +50,22 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 }
 
 #footer {
-  font-size: 10px;
+  height: 20px;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #080808;
+  vertical-align: middle;
+  line-height: 20px;
+  p {
+    text-align: center;
+    margin: 0;
+    font-size: 10px;
+    a{
+      color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.